client/state/automated-transfer/selectors.js

```diff
@@ -68,3 +69,21 @@ export const isEligibleForAutomatedTransfer = compose(
 	getEligibilityStatus,
 	getEligibility
 );
+
+/**
+ * Determine if eligibility warnings or holds exist for
+ * a site other than a business plan hold.
+ *
+ * @param {Object} state global app state
+ * @param {number} siteId requested site for transfer info
+ * @returns {boolean} False if there is one and only one hold/warning referring to missing business plan
+ */
+export const hasEligibilityMessagesOtherThanBusiness = ( state, siteId ) => {
+	const data = getEligibility( state, siteId );
+	const holds = get( data, 'eligibilityHolds', [] );
+	const warnings = get( data, 'eligibilityWarnings', [] );
+
+	return warnings.length > 0 ||
+		holds.length > 1 ||
+		( holds.length > 0 && ! includes( holds, 'NO_BUSINESS_PLAN' ) );
```

Comment:
we can also use an array function designed for analyzing a subset of an array, a projection of an array…

```js
return warnings.length > 0 || difference( holds, [ 'NO_BUSINESS_PLAN' ] ).length > 0 );
```

`difference()` produces a new array without the values given on the right side; in this case it removes the business plan from what we're looking at and then if any other holds remain we know we have a blocker.